### PR TITLE
[Bulk] Broken Link fixes

### PR DESCRIPTION
**Pull request by DhurataJ**
Global effort to fix broken links

@stevemunk

The C+E Skilling team is fixing broken links on learn.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fix/remove and does not change other content. I'd very much appreciate your timely review and feedback on the link changes I've suggested. Thanks!

### CODE REVIEW

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Florianopolis|[Consórcio Fênix](https://www.consorciofenix.com.br) 
 |Fortaleza|[COMPANHIA CEARENSE DE TRANSPORTES METROPOLITANOS](http://www.metrofor.ce.gov.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:64]: I got 

Hmmm… can't reach this page
www.metrofor.ce.gov.br refused to connect.

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Florianopolis|[Consórcio Fênix](https://www.consorciofenix.com.br) 
 |Fortaleza|[COMPANHIA CEARENSE DE TRANSPORTES METROPOLITANOS](http://www.metrofor.ce.gov.br) 
 |Fraiburgo|[Santa Teresinha - Fraiburgo](https://www.santateresinha.com/site/) 
-|Gaspar|[Coletivo Caturani](http://www.coletivocaturani.com.br) 
+|Gaspar|Coletivo Caturani 
 |Indaiatuba|[Sou Indaiatuba](http://www.souindaiatuba.com.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:67]:
```suggestion
|Indaiatuba|[Sou Indaiatuba](https://www.souindaiatuba.com.br) 

```

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Florianopolis|[Consórcio Fênix](https://www.consorciofenix.com.br) 
 |Fortaleza|[COMPANHIA CEARENSE DE TRANSPORTES METROPOLITANOS](http://www.metrofor.ce.gov.br) 
 |Fraiburgo|[Santa Teresinha - Fraiburgo](https://www.santateresinha.com/site/) 
-|Gaspar|[Coletivo Caturani](http://www.coletivocaturani.com.br) 
+|Gaspar|Coletivo Caturani 
 |Indaiatuba|[Sou Indaiatuba](http://www.souindaiatuba.com.br) 
 |Jaragua do Sul|[Viação Canarinho Ltda - Fretamento](http://www.canarinho.com.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:68]: This site does not support https

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Florianopolis|[Consórcio Fênix](https://www.consorciofenix.com.br) 
 |Fortaleza|[COMPANHIA CEARENSE DE TRANSPORTES METROPOLITANOS](http://www.metrofor.ce.gov.br) 
 |Fraiburgo|[Santa Teresinha - Fraiburgo](https://www.santateresinha.com/site/) 
-|Gaspar|[Coletivo Caturani](http://www.coletivocaturani.com.br) 
+|Gaspar|Coletivo Caturani 
 |Indaiatuba|[Sou Indaiatuba](http://www.souindaiatuba.com.br) 
 |Jaragua do Sul|[Viação Canarinho Ltda - Fretamento](http://www.canarinho.com.br) 
 |Jaragua do Sul|[Viação Canarinho Ltda - Guaramirim](http://www.canarinho.com.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:69]: This site does not support https

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 5]
```diff
@@ -93,8 +93,8 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Santa Cruz do Sul|[Consórcio TCS](http://tcsconsorcio.com.br/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:93]:
```suggestion
|Santa Cruz do Sul|[Consórcio TCS](https://tcsconsorcio.com.br/) 

```

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 6]
```diff
@@ -93,8 +93,8 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Santa Cruz do Sul|[Consórcio TCS](http://tcsconsorcio.com.br/) 
 |Sao Paulo|[Empresa Metropolitana de Transportes Urbanos de São Paulo S.A.](https://www.emtu.sp.gov.br/EMTU/home.htm) 
 |Sao Paulo|[SPTRANS](https://www.sptrans.com.br/?versao=221018) 
-|Telemaco Borba|[VINSA](https://www.vinsa.com.br) 
-|Teutonia|[Stadtbus - Teutônia](http://www.stadtbus.com.br/capa/teutonia/itinerario_teutonia.html)
+|Telemaco Borba|[VINSA](http://www.vinsa.com.br) 
+|Teutonia|[Stadtbus - Teutônia](http://www.stadtbus.com.br)
 |Tramandai|[Expresso São José - ANTT](https://www.saojosebus.com.br) 
 |Tramandai|[Expresso São José - Metroplan / Daer](http://www.saojosebus.com.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:99]:
```suggestion
|Tramandai|[Expresso São José - Metroplan / Daer](https://www.saojosebus.com.br) 

```

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 7]
```diff
@@ -93,8 +93,8 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Santa Cruz do Sul|[Consórcio TCS](http://tcsconsorcio.com.br/) 
 |Sao Paulo|[Empresa Metropolitana de Transportes Urbanos de São Paulo S.A.](https://www.emtu.sp.gov.br/EMTU/home.htm) 
 |Sao Paulo|[SPTRANS](https://www.sptrans.com.br/?versao=221018) 
-|Telemaco Borba|[VINSA](https://www.vinsa.com.br) 
-|Teutonia|[Stadtbus - Teutônia](http://www.stadtbus.com.br/capa/teutonia/itinerario_teutonia.html)
+|Telemaco Borba|[VINSA](http://www.vinsa.com.br) 
+|Teutonia|[Stadtbus - Teutônia](http://www.stadtbus.com.br)
 |Tramandai|[Expresso São José - ANTT](https://www.saojosebus.com.br) 
 |Tramandai|[Expresso São José - Metroplan / Daer](http://www.saojosebus.com.br) 
 |Tramandai|[Expresso São José - Rotas Escolares](http://www.saojosebus.com.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:100]:
```suggestion
|Tramandai|[Expresso São José - Rotas Escolares](https://www.saojosebus.com.br) 

```

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 8]
```diff
@@ -158,7 +158,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Gran Concepcion|[Nueva Ruta 160](http://moovitapp.com)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:158]:
```suggestion
|Gran Concepcion|[Nueva Ruta 160](https://moovitapp.com) 

```

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 9]
```diff
@@ -158,7 +158,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Gran Concepcion|[Nueva Ruta 160](http://moovitapp.com) 
 |Gran Concepcion|[Nueva Sol Yet](http://www.gorebiobio.cl/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:159]: This site does not support https

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 10]
```diff
@@ -158,7 +158,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Gran Concepcion|[Nueva Ruta 160](http://moovitapp.com) 
 |Gran Concepcion|[Nueva Sol Yet](http://www.gorebiobio.cl/) 
 |Gran Concepcion|[Nueva Sotrapel](http://www.gorebiobio.cl/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:160]: This site does not support https

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 11]
```diff
@@ -158,7 +158,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Gran Concepcion|[Nueva Ruta 160](http://moovitapp.com) 
 |Gran Concepcion|[Nueva Sol Yet](http://www.gorebiobio.cl/) 
 |Gran Concepcion|[Nueva Sotrapel](http://www.gorebiobio.cl/) 
-|Gran Concepcion|[Nueva Takora](http://www.fesur.cl/es/blog/comenzo-integracion-de-buses-nueva-takora-con-biotren/) 
+|Gran Concepcion|Nueva Takora 
 |Gran Concepcion|[Pedro De Valdivia](http://www.alianzabus.cl/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:162]:
```suggestion
|Gran Concepcion|[Pedro De Valdivia](https://www.alianzabus.cl/) 

```

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 12]
```diff
@@ -158,7 +158,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Gran Concepcion|[Nueva Ruta 160](http://moovitapp.com) 
 |Gran Concepcion|[Nueva Sol Yet](http://www.gorebiobio.cl/) 
 |Gran Concepcion|[Nueva Sotrapel](http://www.gorebiobio.cl/) 
-|Gran Concepcion|[Nueva Takora](http://www.fesur.cl/es/blog/comenzo-integracion-de-buses-nueva-takora-con-biotren/) 
+|Gran Concepcion|Nueva Takora 
 |Gran Concepcion|[Pedro De Valdivia](http://www.alianzabus.cl/) 
 |Gran Concepcion|[Penco Chiguayante Sur](http://www.gorebiobio.cl/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:163]: This site does not support https

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 13]
```diff
@@ -158,7 +158,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Gran Concepcion|[Nueva Ruta 160](http://moovitapp.com) 
 |Gran Concepcion|[Nueva Sol Yet](http://www.gorebiobio.cl/) 
 |Gran Concepcion|[Nueva Sotrapel](http://www.gorebiobio.cl/) 
-|Gran Concepcion|[Nueva Takora](http://www.fesur.cl/es/blog/comenzo-integracion-de-buses-nueva-takora-con-biotren/) 
+|Gran Concepcion|Nueva Takora 
 |Gran Concepcion|[Pedro De Valdivia](http://www.alianzabus.cl/) 
 |Gran Concepcion|[Penco Chiguayante Sur](http://www.gorebiobio.cl/) 
 |Gran Concepcion|[Playa Blanca](http://moovitapp.com)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:164]:
```suggestion
|Gran Concepcion|[Playa Blanca](https://moovitapp.com) 

```

**Review by stevemunk — CHANGES_REQUESTED** [review]
Please make all links to sites that support secure logon start with https

**Comment by DhurataJ** [comment]
Hi @stevemunk, thank you for your feedbacks. While I was working, I noticed that most of the external website domains do not support (secure connection) https. Do you want me to remove those links from articles?

**Comment by stevemunk** [comment]
> Hi @stevemunk, thank you for your feedbacks. While I was working, I noticed as well that most of the external website domains do not support (secure connection) https. Do you want me to remove those links from articles?

@DhurataJ: I tested every link that I commented on, and I marked each with either a suggestion to change it to `https`, or a comment saying that it does not support secure connections (_This site does not support https_). For those that don't support secure connections, just leave them as is. Do not remove the links. Those that do support secure connections need to be changed to `https`.

**Comment by DhurataJ** [comment]
> > Hi @stevemunk, thank you for your feedbacks. While I was working, I noticed as well that most of the external website domains do not support (secure connection) https. Do you want me to remove those links from articles?
> 
> @DhurataJ: I tested every link that I commented on, and I marked each with either a suggestion to change it to `https`, or a comment saying that it does not support secure connections (_This site does not support https_). For those that don't support secure connections, just leave them as is. Do not remove the links. Those that do support secure connections need to be changed to `https`.

Thanks, Steve for following up and clarification in that matter. Suggestions applied to those that needed to be changed to `https`.

**Comment by DhurataJ** [comment]
Thanks @stevemunk for following up and clarification in that matter. Suggestions applied to those that needed to be changed to `https`.

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 14]
```diff
@@ -63,8 +65,9 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Florianopolis|[Consórcio Fênix](https://www.consorciofenix.com.br) 
 |Fortaleza|[COMPANHIA CEARENSE DE TRANSPORTES METROPOLITANOS](http://www.metrofor.ce.gov.br)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/south-america.md:66]: Please remove this link

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 15]
```diff
@@ -27,21 +27,21 @@ Bing Maps APIs provide transit information for the following transit agencies in
...
-|Mendoza|[Gral. Roca S.R.L.](http://generalrocasrl.com.ar) 
+|Mendoza|Gral. Roca S.R.L. 
 |Mendoza|[Mitre S.R.L.](https://www.transportesbmitre.com.ar/) 
 |Mendoza|[Sociedad de Transporte de Mendoza](https://stmendoza.com)
 
 ## Brazil
 |Locality|Transit Agency|
 |:--------------------|--------------|
-|Apucarana|[Viação Apucarana Ltda](https://www.valap.com.br/) 
+|Apucarana|[Viação Apucarana Ltda](http://www.valap.com.br)
```
[the remark this answers is not in the document]
Comment (v-emilypr) [reply BingMaps/coverage/transit-coverage/south-america.md:42]: Confirmed

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 16]
```diff
@@ -51,10 +51,10 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Cabo Frio e Região|[Salineira](http://www.salineira.com.br/) 
 |Cabo Frio e Região|[São Pedro](http://www.salineira.com.br/) 
 |Cachoeirinha|[Stadtbus - Cachoeirinha](http://www.stadtbus.com.br/) 
-|Campina Grande|[PMCG - STTP - Consórcio Santa Maria](https://www.ciomcg.com.br/) 
-|Campina Grande|[PMCG - STTP - Consórcio Santa Verônica](https://www.ciomcg.com.br/) 
+|Campina Grande|PMCG - STTP - Consórcio Santa Maria 
+|Campina Grande|PMCG - STTP - Consórcio Santa Verônica
 |Campinas|[Emdec](http://www.emdec.com.br/) 
-|Campo Bom|[Consórcio Coletivo Campo Bom](http://www.stadtbus.com.br/capa/campo_bom/itinerario_campo_bom.html) 
+|Campo Bom|[Consórcio Coletivo Campo Bom](http://www.stadtbus.com.br)
```
[the remark this answers is not in the document]
Comment (v-emilypr) [reply BingMaps/coverage/transit-coverage/south-america.md:59]: confirmed

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 17]
```diff
@@ -77,8 +77,8 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Londrina|[Transportes Coletivos Grande Londrina](#) 
 |Marilia|[Grande Marília](https://www.amtumarilia.com.br) 
 |Marilia|[Sorriso de Marília](https://www.amtumarilia.com.br) 
-|Maringá|[Cidade Verde Transporte Rodoviário](https://www.cidadeverdetransporte.com.br) 
-|Maringá|[Transporte Coletivo Cidade Canção](https://www.tccc.com.br) 
+|Maringá|[Cidade Verde Transporte Rodoviário](http://www.cidadeverdetransporte.com.br)
```
[the remark this answers is not in the document]
Comment (v-emilypr) [reply BingMaps/coverage/transit-coverage/south-america.md:83]: confirmed

File: BingMaps/coverage/transit-coverage/south-america.md [hunk 18]
```diff
@@ -93,8 +93,8 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Santa Cruz do Sul|[Consórcio TCS](http://tcsconsorcio.com.br/) 
 |Sao Paulo|[Empresa Metropolitana de Transportes Urbanos de São Paulo S.A.](https://www.emtu.sp.gov.br/EMTU/home.htm) 
 |Sao Paulo|[SPTRANS](https://www.sptrans.com.br/?versao=221018) 
-|Telemaco Borba|[VINSA](https://www.vinsa.com.br) 
-|Teutonia|[Stadtbus - Teutônia](http://www.stadtbus.com.br/capa/teutonia/itinerario_teutonia.html)
+|Telemaco Borba|[VINSA](http://www.vinsa.com.br)
```
[the remark this answers is not in the document]
Comment (v-emilypr) [reply BingMaps/coverage/transit-coverage/south-america.md:100]: confirmed

**Review by stevemunk — APPROVED** [review]

**Pull request merged**